### PR TITLE
Support tables up to 128 columns in size

### PR DESCRIPTION
**Pull request by sgrif**
Because @RobertWHurst asked for it. ;)

I think I probably want to up `large-tables` to 32 and rename it, and up
`huge-tables` to 64 and rename it, but that's another PR.

### CODE REVIEW

**Comment by killercup** [comment]
dat diff

**Comment by RobertWHurst** [comment]
Exciting stuff. Thanks for this guys. 😄

File: diesel/Cargo.toml [hunk 1]
```diff
@@ -46,6 +46,7 @@ unstable = []
 lint = ["clippy"]
 large-tables = []
 huge-tables = ["large-tables"]
+128-column-tables = ["huge-tables"]
```
Comment (killercup) [inline diesel/Cargo.toml]: FYI correct feature name would of course be `humongous-tables`

Comment (RobertWHurst) [reply diesel/Cargo.toml]: massive-tables?

**Comment by sgrif** [comment]
screen shot 2018-01-29 at 3 14 59 pm
wait for it...

File: diesel/Cargo.toml (hunk 1 above)
Comment (sgrif) [reply diesel/Cargo.toml]: Let's not start this again lol

**Comment by sgrif** [comment]
It compiles in 9:30 >_<

**Pull request merged**